### ISSUE TITLE
b.e23_alpha16g.BLT1850.ne30_t232.064intel

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#464]
**Purpose:**

Comparing standard compiler (`intel`) and the 'new' intel one (`intel-oneapi`)

Additionally, this run can also be viewed as a perturbation of 64. 
___

**Description:**

Same as run 64 except:
- add `--compiler intel-oneapi` to the `create_newcase` 
___
**Case directory:**
- Locally (if still available):
`/glade/p/cesmdata/cseg/runs/cesm2_0/b.e23_alpha16g.BLT1850.ne30_t232.064intel`

- On github:
https://github.com/NCAR/amwg_dev/tree/b.e23_alpha16g.BLT1850.ne30_t232.064intel
___
**Sandbox:**
- Locally (if still available): `/glade/work/hannay/cesm_tags/cesm2_3_alpha16g`
 
- On github: https://github.com/ESCOMP/cesm/tree/cesm2_3_alpha16g
___
**Diagnostics:**
- Diags (if available):
[https://webext.cgd.ucar.edu/BLT1850/b.e23_alpha16g.BLT1850.ne30_t232.064intel/](https://webext.cgd.ucar.edu/BLT1850/b.e23_alpha16g.BLT1850.ne30_t232.064intel/)
___
**Output:**
- Raw output on derecho (if still available): `/glade/derecho/scratch/hannay/archive/b.e23_alpha16g.BLT1850.ne30_t232.064intel/`
- CAM Climos: `/glade/campaign/cgd/amp/amwg/climo/b.e23_alpha16g.BLT1850.ne30_t232.064intel`
___
**Contacts:**
@cecilehannay @jedwards4b, @briandobbins
___
**Extra details:**

**Comment by adamrher** [comment]
@briandobbins this new intel compiler run (red) is indistinguishable from the any of the 64 series runs. The 64 series runs are just pertlim experiments from the original run 64. The grey thick line is a ~2 year old case base, which does have a different climate, for reference. Though I think the ocean may have been initialized differently in 26g. 

@cecilehannay It can't hurt to run ADF on the first twenty years of 64intel and 64, so we can compare the fields in a lat-lon plot.

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/a462dc88-00a1-4790-9145-c4c6f978d9ae)

**Comment by cecilehannay** [comment]
@adamrher: I had already done this comparison. https://webext.cgd.ucar.edu/BLT1850/b.e23_alpha16g.BLT1850.ne30_t232.064intel/atm/
and for me they are the same climate.

**Comment by jedwards4b** [comment]
@briandobbins Do you think we should run this one longer?  I am satisfied that there are no significant differences from ifort to ifx.   The only problem is with performance, the ifx run is just over 1ypd slower than ifort.

**Comment by briandobbins** [comment]
@adamrher @cecilehannay  Great, thank you both so much.  This was the hoped-for result.

@jedwards4b I don't think so right now.  I think down the line, we may do a short run (5 year?) ensemble to ensure there isn't a statistically significant difference in variance or mean, but that doesn't seem urgent given the above.  It's only a consideration given that ifort is EOL soon, and thus it could necessitate a change in compilers during experiments in the future.  Happy to hear if you think that's unnecessary, but given the inexpensive cost of a short run it seems worth it to me for the peace of mind.

**Comment by adamrher** [comment]
Sorry @cecilehannay I misunderstood. I didn't realize you already did the ADF comparison. The climate is identical.

@jedwards4b there's 30+ timing files each with a SYPD estimate. Is that enough?

no problemo @briandobbins

**Comment by jedwards4b** [comment]
@adamrher yes we can stop this one - we will continue to watch development of the new ifx compiler and hopefully they will be able to close this performance gap before they end ifort support.
